src/App.tsx

```diff
@@ -15,33 +14,75 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <div className="App">
-    <button type="button">
-      Start
-    </button>
+export const App: React.FC = () => {
+  const [listIsVisible, setListVisibility] = useState(false);
+  const [goods, setGoods] = useState([...goodsFromServer]);
```

Comment:
let's better add the flags `isReversed` and `sortType`, just like we did in previous task, and sort our `goodsFromServer` array based on it, without saving it to the state
```suggestion
```